Fix "Visit website" button crashes app if there is no browser installed

Xeitor opened the pull request:
Changed VisitWebsiteActionButton onClick method to use IntentUtils class.

Closes #4507

ByteHamster commented:
Thanks! This bug fix can go directly to the `master` branch (for version 2.0.0) and does not need to wait for 2.1.0. You can change the branch with something like the following:

```
git reset --hard master
git cherry-pick 75a8afd
git push --force
```

After that, the target branch on GitHub can be changed using the edit button on the pull request page.

Xeitor commented:
Hey @ByteHamster, did  I do things right ? I changed branch to master locally, pushed changes and changed the branch on the pull request to master.

ByteHamster commented:
> did I do things right?

Did you try the commands I outlined above? It sounds like you just checked out the master branch but you did not change the history in the branch that you pushed from (`develop`). Your change is still added on top of the `develop` branch and therefore contains a lot of unrelated changes (as you can see on GitHub). To reset your `develop` branch to master and then only apply the change on top of that one, you can use the commands above.

Xeitor commented:
Could not manage to solve the issue, I am new to Github's workflow I am currently tryng to get my head arround everything. Sorry for the drawbacks.

ByteHamster commented:
Replaced with #4520
